$MirOS: ports/mailnews/pine/patches/patch-imap_src_osdep_unix_phile_c,v 1.1.7.1 2005/03/18 15:49:48 tg Exp $
--- imap/src/osdep/unix/phile.c.orig	Tue Apr 27 20:00:25 2004
+++ imap/src/osdep/unix/phile.c	Wed May  4 23:28:08 2005
@@ -289,8 +289,9 @@ MAILSTREAM *phile_open (MAILSTREAM *stre
   elt->zoccident = (k == i) ? 0 : 1;
   elt->zhours = k/60;
   elt->zminutes = k % 60;
-  sprintf (tmp,"%s, %d %s %d %02d:%02d:%02d %c%02d%02d",
-	   days[t->tm_wday],t->tm_mday,months[t->tm_mon],t->tm_year+1900,
+  sprintf (tmp,"%s, %d %s %ld %02d:%02d:%02d %c%02d%02d",
+	   days[t->tm_wday],t->tm_mday,months[t->tm_mon],
+	   (long)t->tm_year+1900,
 	   t->tm_hour,t->tm_min,t->tm_sec,elt->zoccident ? '-' : '+',
 	   elt->zhours,elt->zminutes);
 				/* set up Date field */
